src/energyhub.py

```diff
@@ -502,8 +510,19 @@ def _solve_monte_carlo(self, objective):
             self._monte_carlo_set_cost_parameters()
             if run == 0:
                 self._optimize(objective)
+                self.write_results()
             else:
                 self._call_solver()
+                self.write_results()
+
+        summary_path = Path.joinpath(
+            Path(config["reporting"]["save_summary_path"]["value"]), "Summary.xlsx"
+        )
+        if config["optimization"]["monte_carlo"]["type"]["value"] == 1:
+            component_set = config["optimization"]["monte_carlo"]["on_what"]["value"]
```

Comment:
Can we make this not an integer but a string to be more descriptive?

Reply:
1: "normal distribution"
2: "uniform distribution  from file"

Reply:
done